TradeItIosTicketSDK2Tests/TradeItLinkedBrokerCacheSpec.swift

```diff
@@ -24,12 +23,12 @@ class TradeItLinkedBrokerCacheSpec: QuickSpec {
         describe("TradeItLinkedBrokerCache") {
             beforeEach {
                 // This doesn't work in the simulator.  Thanks, Apple.
-                self.defaults.removeAll()
+                // self.defaults.removeSuite(named: "TRADE IT TEST SUITE")
```

Comment:
Or was it a SwiftyUserDefaults thing? Don't know. Removing, but leaving the comment because it seems like it will save someone heartache.